LICENSE_THIRDPARTY

```diff
@@ -0,0 +1,1603 @@
+-------------------------------------------------------------------------------------
```

Comment:
@sharkdp you said in #324:

> I just meant: I think we're not required to attribute the dependencies in the repository itself, only in the npm package and in the web version. We still could it of course. Maybe that would be the easiest option anyway.. if we don't automatically build this attribution-file during deployment, but have it checked in to the repo itself. On the other hand, that would probably just make it harder because we would then need to make sure it's always up-to-date.

What do you think about this now? I agree that it's easier to just have the files in the repo, and since the CI would check that it's always up-to-date, I think it shouldn't be an issue.

Reply:
Yes. If we make sure it's up-to-date, having it in the repo seems fine to me.